plugins/products.ts

```diff
@@ -35,9 +35,9 @@ function configureClient() {
     };
 }
 
-function loadRemoteModules() {
+function loadRemoteModules(): ActionFunc<boolean> {
     /* eslint-disable no-console */
-    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+    return async (dispatch, getState) => {
```

Comment:
To double check, does this define the types of `dispatch` and `getState` correctly? I was running into issues where they were both `any` when I tried something similar